packages/mesa/build.sh

```diff
@@ -22,14 +23,26 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 -Dgles1=disabled
 -Dgles2=enabled
 -Ddri3=enabled
+-Ddri-drivers=[]
 -Dglx=dri
--Dllvm=enabled
+-Dllvm=disabled
```

Comment:
Can't llvm be enabled? This could lead to some performance degradation.

Reply:
Yes